packages/components/src/components/dropdown/dropdown.test.ts

```diff
@@ -144,10 +144,6 @@ describe('<sd-dropdown>', () => {
     expect(panel.hidden).to.be.true;
   });
 
-  /**
-   * TODO: Re-enable this test once sd-menu and sd-menu-item are implemented
-   */
-
   // it('should still open on arrow navigation when no menu items', async () => {
```

Comment:
what about those tests now?

Reply:
I created a follow-up ticket to either remove logic + tests regarding `sd-menu` or build the component: https://github.com/solid-design-system/figma/issues/111